core/model/src/main/java/org/activityinfo/model/type/attachment/AttachmentType.java

```diff
@@ -68,22 +71,38 @@ public FormClass getParameterFormClass() {
             cardinalityType.getValues().add(new EnumItem(ResourceId.valueOf("single"), "Single"));
             cardinalityType.getValues().add(new EnumItem(ResourceId.valueOf("multiple"), "Multiple"));
 
+            EnumType kindType = (EnumType) EnumType.TYPE_CLASS.createType();
+            kindType.getValues().add(new EnumItem(ResourceId.valueOf(Kind.ATTACHMENT.name()), "Attachment"));
+            kindType.getValues().add(new EnumItem(ResourceId.valueOf(Kind.IMAGE.name()), "Image"));
+
             FormClass formClass = new FormClass(ResourceIdPrefixType.TYPE.id("image"));
             formClass.addElement(new FormField(ResourceId.valueOf("cardinality"))
                     .setType(cardinalityType)
                     .setLabel("Cardinality")
                     .setDescription("Determines whether users can add a single image, or multiple images")
             );
+            formClass.addElement(new FormField(ResourceId.valueOf("kind"))
+                            .setType(kindType)
+                            .setLabel("Type")
+                            .setDescription("Determines type of attachment (image or file)")
+                            .setVisible(false)
+            );
             return formClass;
         }
     }
 
     public static final TypeClass TYPE_CLASS = new TypeClass();
 
+    public enum Kind {
```

Comment:
Why not just parametrize on mime type? Images could have the mimetype `image/*`, otherwise any attachment would be "*/*".